docker/service/Dockerfile

```diff
@@ -12,7 +12,7 @@ RUN mv -v search-service/target/search-service-*.jar app.jar \
 # Clean up image so we can rerun without the cache being invalidated because target files are changed
     && mvn clean
 
-FROM openjdk:11-jre
+FROM eclipse-temurin:17
```

Comment:
Only include the jre. See register spring-boot-base/Dockerfile.

Reply:
Mmm but this is a `gwt-lib`, it's just packaging the java in a special way no?